Fachportal Padagogik.js

```diff
@@ -0,0 +1,313 @@
+				"volume": "7",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"Deutschland",
+					"Empirische Untersuchung",
+					"Fragebogen",
+					"Lehrer",
+					"Lernförderung",
+					"Lernmethode",
+					"Mathematikunterricht",
+					"Schuljahr 09",
+					"Selbstgesteuertes Lernen",
+					"Unterrichtsforschung",
+					"Überzeugung"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.fachportal-paedagogik.de/fis_bildung/suche/fis_set_e.html?FId=A18490&mstn=3&ckd=no&mtz=100&facets=y&maxg=5&suche=erweitert&ohneSynonyme=y&feldname1=Freitext&feldinhalt1=MATHEMATIK&bool1=and&nHits=19233&next=A18569,A19036,1065678&prev=A18195,A18196&marker=1",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "10 Jahre international vergleichende Schulleistungsforschung in der Grundschule. Vertiefende Analysen zu IGLU und TIMSS 2001 bis 2011.",
+				"creators": [
+					{
+						"firstName": "Heike ",
+						"lastName": "Wendt",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Tobias C. ",
+						"lastName": "Stubbe",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Knut ",
+						"lastName": "Schwippert",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Wilfried",
+						"lastName": "Bos",
+						"creatorType": "editor"
+					}
+				],
+				"date": "2015",
+				"ISBN": "9783830933335",
+				"abstractNote": "\"Regelmäßige Schulleistungsstudien erfassen Stärken und Schwächen des Bildungswesens und geben Hinweise für gezielte Maßnahmen zur Qualitätsverbesserung. Die Internationale Grundschul-Lese-Untersuchung (IGLU) findet seit 2001 alle fünf Jahre statt und richtet den Fokus auf die Lesekompetenz von Schülerinnen und Schülern am Ende der Grundschulzeit. An der Trends in International Mathematics and Science Study (TIMSS) im Grundschulbereich, die alle vier Jahre die Mathematik- sowie die Naturwissenschaftskompetenz beleuchtet, beteiligt sich Deutschland seit 2007. 2011 wurden IGLU und TIMSS erstmals parallel durchgeführt, daher können hier vertiefende Analysen beider Studien zusammengeführt werden. Zudem liegen mit der dritten Beteiligung an IGLU Trenddaten vor, die es erlauben, Entwicklungen der Grundschule in Deutschland der letzten zehn Jahre nachzuzeichnen.\" [Zusammenfassung: Angaben des Autors der Webseite].;;;This book analyses 10 years (2001-2011) of international assessment studies with TIMSS (Trends in International Mathematics and Science Study) and PIRLS (Progress in International Reading Literacy Study) with focus on Germany and Europe. [Abstract: Editors of Education Worldwide]",
+				"itemID": "book",
+				"libraryCatalog": "Fachportal Pädagogik",
+				"numPages": "262 S.",
+				"place": "Münster",
+				"publisher": "Waxmann",
+				"url": "https://www.waxmann.com/index.php?id=buecher&no_cache=1&tx_p2waxmann_pi1[oberkategorie]=OKA999999&tx_p2waxmann_pi1[buch]=BUC124353",
```

Comment:
we don't want the URL -- assuming the catalog has no full texts, either strip URLs entirely or move them into links with `snapshot: false`